dev/io.openliberty.jaxws.config_fat/test-applications/simpleTestService/src/io/openliberty/jaxws/fat/stubclient/client/SimpleEchoService.java

```diff
@@ -30,9 +30,19 @@ public class SimpleEchoService extends Service {
         try {
             URL baseUrl;
             baseUrl = io.openliberty.jaxws.fat.stubclient.client.SimpleEchoService.class.getResource(".");
-            url = new URL(baseUrl, new StringBuilder().append("http://localhost:").append(Integer.getInteger("bvt.prop.HTTP_default")).append("/simpleTestService/SimpleEchoService?wsdl").toString());
-            } catch (MalformedURLException e) {
-            logger.warning("Failed to create URL for the wsdl Location: " + new StringBuilder().append("http://localhost:").append(Integer.getInteger("bvt.prop.HTTP_default")).append("/simpleService/SimpleEchoService?wsdl").toString()+ ", retrying as a local file");
+
+            String host = System.getProperty("hostName");
+            if (host == null) {
+                logger.info("Failed to obtain host from system property, hostName, falling back to localhost");
+                host = "localhost";
+            }
+
+            url = new URL(baseUrl, new StringBuilder().append("http://" + host
+                                                              + ":").append(Integer.getInteger("bvt.prop.HTTP_default")).append("/simpleTestService/SimpleEchoService?wsdl").toString());
+        } catch (MalformedURLException e) {
+            logger.warning("Failed to create URL for the wsdl Location: "
+                           + new StringBuilder().append("http://localhost:").append(Integer.getInteger("bvt.prop.HTTP_default")).append("/simpleService/SimpleEchoService?wsdl").toString()
```

Comment:
The host in in the warning message might not be same as generated in url. Is there a specific reason why we are not creating String portion of URL and use on both creation of URL and in warning message?